backend/box/box.go

```diff
@@ -246,6 +251,7 @@ type Options struct {
 	CommitRetries int                  `config:"commit_retries"`
 	Enc           encoder.MultiEncoder `config:"encoding"`
 	RootFolderID  string               `config:"root_folder_id"`
+	AccessToken   string               `config:"box_primary_access_token"`
```

Comment:
Can you remove the `box_` from this otherwise the generated config flags will stutter `--box-box-primary-access-token`

Would calling it `access_token` be OK? Just to shorten it a bit?

Reply:
Oh yeah, I'll remove `box_`. There's a "Secondary Access Token" in the box admin panel which is used to roll over the primary access token. Wasn't sure if people would get them confused. I guess I could change it to `access_token` and rely on people reading the help message?